package.json

```diff
@@ -69,6 +69,7 @@
     "@types/stellar-base": "^0.10.2",
     "bignumber.js": "^8.1.1",
     "change-case": "^3.1.0",
+    "js-sha1": "^0.6.0",
```

Comment:
I thought about adding this as `peerDependency` and only using it when node's crypto and browser's `crypto.subtle` are not available. Not sure it's worth it.